### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by Volcro** [NateScarlet/auto-derby#251] [Q&A] [answered]
Hi:

第一次遇到的事件需要人工选择(選項順序由上而下123... 在cmd中輸入數字即可)
预测名次低于设定值，暂停等待学技能

想问一下这两种情况如果像写死选1/不学技能该修改哪些个文件，thxx

**Comment by NateScarlet** [comment, marked as answer]
不需要写死选1，如果想避免操作 用 [afk 插件](https://github.com/NateScarlet/auto-derby/wiki/Plugins#afk)就行

**Reply by Volcro** [reply]
好的，thxx
